packages/ava/src/advisor/ruler/types.ts

```diff
@@ -0,0 +1,143 @@
+// eslint-disable-next-line import/no-unresolved
+import type { DataTypes } from '../../data';
+import type { CkbTypes } from '../../ckb';
+import type { Specification } from '../../common/types';
+
```

Comment:
plz modify this file to `rename` instead of `new file`

Reply:
done